src/integrators/sdirk_integrators.jl

```diff
@@ -72,15 +67,21 @@ end
 
   cache.ηold = η
   cache.newton_iters = iter
-  u = uprev + z
+  u = u .+ dz
```

Comment:
If floating point works correctly then yes. There's a good chance this adds a few small values in the end of the Newton iteration though if the equation is stiff enough and the values are large enough. In that case, repeat adding of `dz` to `u` will accumulate errors. It's fine to "accumulate errors` onto `z` because that's the iteration variable, so it will iterate more if `z` is sufficiently incorrect. But accumulating into `u` which is then called in `f` can lead to `f` being off, which will cause problems in stiff problems. In the end, this accumulation doesn't make `u` exactly satisfy the tableau which can lead to order loss.